Makefile

```diff
@@ -60,6 +63,9 @@ else
 	glide install
 endif
 
+.PHONY: verify-dep
+verify-dep:
```

Comment:
I’d suggest doing away with the separate target, it doesn’t provide much value vs inline command